apps/test/unit/lib/levelbuilder/lesson-editor/LevelTokenDetailsTest.js

```diff
@@ -51,20 +51,37 @@ describe('LevelTokenDetails', () => {
       activitySectionPosition: 5,
       activityPosition: 1,
       lessonExtrasAvailableForUnit: false,
+      isProfessionalLearningCourse: false,
       inactiveLevelNames: []
     };
   });
 
   it('renders with default props', () => {
     const wrapper = shallow(<LevelTokenDetails {...defaultProps} />);
 
-    assertCheckboxVisible(wrapper, 'bonus', true);
-    assertCheckboxVisible(wrapper, 'assessment', true);
-    assertCheckboxVisible(wrapper, 'challenge', true);
+    assertCheckboxVisible(wrapper, 'Bonus', true);
```

Comment:
I know that "renders with default props" was encouraged at one point, but can you please change the name of this test case to describe what is being tested? e.g. it "renders unchecked checkboxes"